### ISSUE TITLE
Unable to send Miota to my bitfinix account

### ISSUE THREAD

**Issue by zinnyko** [iotaledger/wallet#619]
I have been trying since yesterday but i am unable to send my miota to my bitfinix account, plz hep
Here the hash for the transaction: https://iotasear.ch/address/NMBAZXHAFRQXUZNRPNLSLWRBZEYECNDNUJY9QIBC9FVJYRFBWWLY9C9IVDPFIRHKTHZYWASXTZSAKZAVDPASLJOLKC

i reattached multiple times as i thought that it is the only way but even thats not working.

**Comment by mnori** [comment]
I'm having the same problem. Yesterday I made a test transaction to a different wallet. I reattached multiple times but the thing is still stuck on pending. Also thanks to my test transaction, it's impossible to make any more transactions until the original one confirms. Maybe there's some issue with the network? 

Doesn't inspire much confidence anyway. I'll probably be dumping most of my coins as soon as I can get them out of the wallet. I guess maybe that's why the price is so high? People are getting excited over the price increase, buying, and then being unable to dump later and take profit, driving the price ever higher. Just a theory

**Comment by CryptoIOTA** [comment]
Similar issue: I get a "Request error' when attempting to send to my Bitfinex account. I'm running Iota light wallet 2.5.4 on Win 10. Tried with different nodes.

Questions:
1) do I really need to attach the receiving address to tangle even if I'm sending? I tried with different nodes, but always get "Request Error"
2) Shoul I try any particular node (I tried a few from iota.dance/nodes).
3) could this be a general issue with the tangle?

Thank you for any help!

**Comment by CryptoIOTA** [comment]
I was eventually able to attach the receiving address to tagnle using node02.iotatoken.nl:14265 with CCurl Implementation.

But sending iota to Bitfinex sill fails with "Request error".
What is wrong with this thing???

**Comment by tozo07** [comment]
I have same problem.
I tried meny times:
- Reattach
- Rebroadcast

**Comment by mnori** [comment]
My transaction eventually confirmed after around 20 hours of waiting.

**Comment by prodigydefender** [comment]
I tested sending 1 Miota to Bitfinex, got a confirmation screen on my IOTA wallet, but it hasn't gotten to my Bitfinex wallet yet, and it has been almost 48 hrs.

**Comment by prodigydefender** [comment]
Additional comment on sending Miota to Bitfinex is that you have to edit node config to iota.bitfinex.com:80

**Comment by tozo07** [comment]
I have a 6 day old Pending.
https://iotasear.ch/address/VLLOKT9WGIMNMIUPMFQ9KXZ9CDSSNUP9TVODKJPVAF9HGZFMGVWHDMJNEBFKNTHXJUYXUYGIZPZCGQPOXKIZBUYVHY

**Comment by Kshysiek** [comment]
The same problem here. Reattaching does not help.

**Comment by gitdisrupt** [comment]
Any luck here? I believe my account was HACKED in an attempt to send a small amount to bitfinex. INCREDIBLY upset.

**Comment by CryptoIOTA** [comment]
No news on my side. Still getting "Request Error" when attempting to send few IOTAs to bitfinex.
The transaction is not even being sent in my case as I don't see it in my wallet's history.
Tried also to uninstall/reinstall the wallet.

**Comment by CryptoIOTA** [comment]
Just an update on the "Request Error" when attempting to send IOTA: I was eventually able to perform the transaction using installing the wallet on a different PC. Still don't know why it doesn't work on mine.

Anyway I'm all set now, just waiting for the transaction to be confirmed (it has been pending for 2 hours now!).

**Comment by gitdisrupt** [comment]
I am in the same situation ... i THINK my account is OK, but i wont know till all the TXs have confirmed ... i have been waiting 6+ hours now.

**Comment by kons233** [comment]
same problem with the pending, hey @mnori how many times did you reatach and rebroadcast? how often and how far apart in time till you got it to work? since I see others are waiting days and you say 20 hours you must have done something right

**Comment by mnori** [comment]
I was only sending to a test wallet, not Bitfinex. I attached in the evening and then clicked it a few times in the morning the following day.

I've recently sent a test transaction to Bitfinex and this one's taking longer.

**Comment by mnori** [comment]
Update: My test transaction to Bitfinex is now confirmed, according to the wallet. But it's not showing up in Bitfinex.

**Comment by CryptoIOTA** [comment]
@mnori: same for me. I've finally been able to confirm the transaction by reattaching it but it doesn't show up in Bitfinex.

**Comment by mnori** [comment]
@CryptoIOTA My confirmed IOTA transaction is still invisible on Bitfinex.

Interestingly though, I made another test transaction to Binance and it got confirmed after a mere 1 hour!

**Comment by kons233** [comment]
@mnori how many times would you say you mashed the reattach button, cuz I sent to binance as well about 18 hours ago, and again 9 hours ago everything is still pending, and ive reatached a couple times too but maybe not enough? idk. Also it wouldnt send at all until i switched to regular curl, and also im on a lite node. are you using lite or full? also are you willing to tell me what a bigger transaction is? i have been sending 5, 4, 3, 2 iota at a time i wonder if the bigger the faster?

**Comment by mnori** [comment]
@kons233 Here's some params you might find useful

I hit reattach 4 times in the first 5 minutes
Host: node01.iotatoken.nl (I also tried the other numbers in the series)
Light wallet
Using CCurl
My test transactions were between 1-4 Mi

Good luck!

**Comment by kons233** [comment]
Thanks @mnori Just did exactly that hopefully it gets trough!

**Comment by prodigydefender** [comment]
sent IOTA to bitfinex, been 3 days, still haven't received

**Comment by CryptoIOTA** [comment]
Today transaction got confirmed within 10/15 minutes and I saw my iotas in Bitfinex after 3 hours.
Also I had to disable my windows firewall to fix all the issue I was experiencing the previous day.
Not 100% sure this was the root cause though.

**Comment by prodigydefender** [comment]
had to disable my windows firewall - wouldn't that leave you with
vulnerability on cyber security?

**Comment by CryptoIOTA** [comment]
Yes, I was vulnerable for 15 mins, but it's really unlikely to have someone expliting any open port on that short timeframe. That said, my seed is anyway encrypted in a password manager installed on a USB key.

Next time I'll try to keep the firewall on as I'm not 100% sure it was the root cause of the issue I had the previous day.
